ocaml/sdk-gen/go/templates/SessionMethod.mustache

```diff
@@ -0,0 +1,64 @@
+{{#messages}}
+// {{method_name_exported}}:{{#description}} {{.}}{{/description}}
+{{#has_error}}
+//
+// Errors:
+{{/has_error}}
+{{#errors}}
+// {{name}} - {{doc}}
+{{/errors}}
+func (class *Session) {{method_name_exported}}({{#header_params}}{{^first}}, {{/first}}{{#session}}session *Session{{/session}}{{^session}}{{name_internal}} {{type}}{{/session}}{{/header_params}}) ({{#result}}retval {{type}}, {{/result}}err error) {
```

Comment:
The "session" here seems not necessary?

Reply:
We do different special handlings for `Ref "session"` in two places.